readme.md

```diff
@@ -359,6 +359,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 - [pretty-error](https://github.com/AriaMinaei/pretty-error) - Errors with less clutter.
 - [humanize](https://github.com/taijinlee/humanize) - Data formatter for human readability.
 - [node-read](https://github.com/bndr/node-read) - Extract readable content from any page.
+- [read-art](https://github.com/Tjatse/node-readability) - Scrape/Crawl article from any site automatically. Make any web page readable, no matter Chinese or English.
```

Comment:
@sindresorhus That's a good question and you're really a meticulous owner, that's nice :p
I've worked on `node-read` from the initial phase - to help improving the performance and making it more stabler, you can read from [node-read's PRs](https://github.com/bndr/node-read/pulls?q=is%3Apr+is%3Aclosed), and I mentioned that in the [read-art md](https://github.com/Tjatse/node-readability#bndrnode-read):

> bndr/node-read

>I've contributed on this for a while, but it's hard to communicate with Vadim(we are in a different  timezone), and we have very different ideas. So I decided to write it on my own.

But to be honest, it still can not be used in a production env, lack of:
- Fit such a wide variety of domains (different URLs)
- Read speed
- Memory performance
- A fair mechanism for judging content

I've reconstructed the readability from Arc90's, and made a hight improvement:
- Faster Than Any Readability Module
- High Performance - Less memory
- Automatic Read Title & Content
- Follow Redirects
- Automatic Decoding Content Encodings(Avoid Messy Codes, Especially Chinese)
- Gzip/Deflate Support
- Proxy Support
- Generate User-Agent
- Different outputs: JSON, TEXT and HTML.

More important is that I am using `read-art` in the production - Spider/Crawl title && content from millions sites (only count domains). And it's very stable, quick and efficient.
![data](https://cloud.githubusercontent.com/assets/1418444/5593796/03624d76-9263-11e4-92b8-6b1671b5394d.jpg)
![body](https://cloud.githubusercontent.com/assets/1418444/5593797/03660736-9263-11e4-965b-9cb4a3cb7c5d.jpg)